sycl/include/sycl/ext/oneapi/device_global/properties.hpp

```diff
@@ -19,13 +20,20 @@ namespace sycl {
 inline namespace _V1 {
 namespace ext::oneapi::experimental {
 
-template <typename T, typename PropertyListT> class device_global;
+template <typename T, typename PropertyListT = empty_properties_t,
+          typename = void>
+class device_global;
 
 struct device_image_scope_key
     : detail::compile_time_property_key<detail::PropKind::DeviceImageScope> {
   using value_t = property_value<device_image_scope_key>;
 };
 
+struct device_constant_key
+    : detail::compile_time_property_key<detail::PropKind::DeviceConstant> {
+  using value_t = property_value<device_constant_key>;
+};
```

Comment:
Do we have this new property and its effects on the `device_global` class documented anywhere?